### PR TITLE
Prepare 5.5.1 release

### PR DESCRIPTION
**Pull request by nikku**
Related to https://github.com/camunda/camunda-modeler/issues/3307

### CODE REVIEW

**Review by marstamm — APPROVED** [review]

File: CHANGELOG.md [hunk 1]
```diff
@@ -6,6 +6,13 @@ All notable changes to the [Camunda Modeler](https://github.com/camunda/camunda-
 
 ___Note:__ Yet to be released changes appear here._
 
+## 5.5.1
+
+### BPMN
+
+* `FIX`: ensure `ImplementationProps` do not remove empty properties ([#3303](https://github.com/camunda/camunda-modeler/issues/3303), [#3304](https://github.com/camunda/camunda-modeler/issues/3304))
+* `DEPS`: update to `bpmn-js-properties-panel@1.1.2`
```
Comment (barmac) [inline CHANGELOG.md:14]:
```suggestion
* `DEPS`: update to `bpmn-js-properties-panel@1.11.2`
```

File: THIRD_PARTY_NOTICES [hunk 2]
```diff
@@ -131,15 +131,15 @@ This project incorporates components from the projects listed below. The origina
 124.  @bpmn-io/element-templates-validator@0.11.0 (https://github.com/bpmn-io/element-templates-validator)
 125.  @bpmn-io/extract-process-variables@0.6.0 (https://github.com/bpmn-io/extract-process-variables)
 126.  @bpmn-io/feel-editor@0.4.1 (https://github.com/bpmn-io/feel-editor)
-127.  @bpmn-io/properties-panel@0.23.0 (https://github.com/bpmn-io/properties-panel)
+127.  @bpmn-io/properties-panel@0.24.0 (https://github.com/bpmn-io/properties-panel)
```
Comment (barmac) [inline THIRD_PARTY_NOTICES:134]: Is this not updated automatically with the release?

Comment (nikku) [reply THIRD_PARTY_NOTICES:134]: Yes. It is an automatic update commit, in fact.

**Comment by nikku** [comment]
Cleaned up + incorporated review hint. Thanks!

**Pull request merged**